core/src/main/java/org/mule/component/BindingInvocationHandler.java

```diff
@@ -75,7 +74,9 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
         MuleMessage message = createMuleMessage(args);
 
         // Some transports such as Axis, RMI and EJB can use the method information
-        message.setInvocationProperty(MuleProperties.MULE_METHOD_PROPERTY, method.getName());
+
+        // TODO MULE-9449 Remove Java Component Bindings
+        //message.setInvocationProperty(MuleProperties.MULE_METHOD_PROPERTY, method.getName());
```

Comment:
In order to remove all bindings code in a single PR separately rather than contaminating this commit with other changes.  Rather than delete part of the code and not the rest, decided it was best to comment with a TODO.